core/controllers/admin.py

```diff
@@ -854,3 +854,26 @@ def post(self):
             self.render_json({})
         else:
             raise self.InvalidInputException('This app cannot send emails.')
+
+
+class UpdateUsernameHandler(base.BaseHandler):
+    """Handler for renaming usernames."""
+
+    @acl_decorators.can_access_admin_page
+    def put(self):
+        current_username = self.payload.get('current_username', None)
+        new_username = self.payload.get('new_username', None)
+        if new_username is not None:
+            if len(new_username) > 30:
+                raise self.InvalidInputException(
+                    'Please make sure that the new username is not longer '
+                    'than 30 characters.')
```

Comment:
Use the named constant here too.

In general, always use named constants for numbers, to ensure consistency and a single point of reference.

Reply:
Done.